examples/src/main/java/com/mapbox/navigation/examples/core/RouteAlertsActivity.kt

```diff
@@ -323,7 +351,10 @@ class RouteAlertsActivity : AppCompatActivity() {
         )
     }
 
-    private fun generateSymbolLayerProperties(imageId: String, textId: String) = arrayOf(
+    private fun generateSymbolLayerProperties(
+        imageId: String,
+        textId: String
+    ) = arrayOf(
```

Comment:
```suggestion
    ) : Array<PropertyFactory> = arrayOf(
```

Reply:
It's `private` but will add the explicit type 👍

Reply:
In fact, it's `Array<PropertyValue<out Any>>` 😅